Extensions/TweenBehavior/JsExtension.js

```diff
@@ -0,0 +1,532 @@
+        _("Tween _PARAM2_ on _PARAM0_ has finished playing"),
+        _(""),
+        "JsPlatform/Extensions/tween_behavior24.png",
+        "JsPlatform/Extensions/tween_behavior32.png"
+      )
+      .addParameter("object", _("Object"), "", false)
+      .addParameter("behavior", _("Behavior"), "TweenBehavior", false)
+      .addParameter("string", _("Tween Identifier"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("hasFinished");
+
+    behavior
+      .addAction(
+        "PauseTween",
+        _("Pause a tween"),
+        _("Pause the running tween animation."),
+        _("Pause the tween _PARAM2_ on _PARAM0_"),
+        _(""),
+        "JsPlatform/Extensions/tween_behavior24.png",
+        "JsPlatform/Extensions/tween_behavior32.png"
+      )
+      .addParameter("object", _("Object"), "", false)
+      .addParameter("behavior", _("Behavior"), "TweenBehavior", false)
+      .addParameter("string", _("Tween Identifier"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("pauseTween");
+
+    behavior
+      .addAction(
+        "StopTween",
+        _("Stop a tween"),
+        _("Stop the running tween animation."),
+        _("Stop the tween _PARAM2_ on _PARAM0_"),
+        _(""),
+        "JsPlatform/Extensions/tween_behavior24.png",
+        "JsPlatform/Extensions/tween_behavior32.png"
+      )
+      .addParameter("object", _("Object"), "", false)
+      .addParameter("behavior", _("Behavior"), "TweenBehavior", false)
+      .addParameter("string", _("Tween Identifier"), "", false)
+      .addParameter("yesorno", _("Jump to end"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("stopTween");
+
+    behavior
+      .addAction(
+        "ResumeTween",
+        _("Resume a tween"),
+        _("Resume the tween animation."),
+        _("Resume the tween _PARAM2_ on _PARAM0_"),
+        _(""),
+        "JsPlatform/Extensions/tween_behavior24.png",
+        "JsPlatform/Extensions/tween_behavior32.png"
+      )
+      .addParameter("object", _("Object"), "", false)
+      .addParameter("behavior", _("Behavior"), "TweenBehavior", false)
+      .addParameter("string", _("Tween Identifier"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("resumeTween");
+
+    behavior
+      .addAction(
+        "RemoveTween",
+        _("Remove a tween"),
+        _("Remove the tween animation from the object."),
+        _("Remove the tween _PARAM2_ from _PARAM0_"),
+        _(""),
+        "JsPlatform/Extensions/tween_behavior24.png",
+        "JsPlatform/Extensions/tween_behavior32.png"
+      )
+      .addParameter("object", _("Object"), "", false)
+      .addParameter("behavior", _("Behavior"), "TweenBehavior", false)
+      .addParameter("string", _("Tween Identifier"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("removeTween");
+
+    behavior
+      .addExpression(
+        "GetProgress",
```

Comment:
Convention is that other expression don't have `Get`, so let's just name this "Progress". It's less verbose and obvious that you're using a getter:

`MyObject.Tween::GetProgress()` vs `MyObject.Tween::Progress()`

Reply:
Changed it to "Progress" in JsExtension.js but left the function name "getProgress" in tweenruntimebehavior.js since it is not a property.

Reply:
function name getProgress is fine as it's the JavaScript interface - in which getXXX and setXXX is good practice :) It's just on GDevelop side that we want to avoid GetXXX in favor of XXX, as all expressions are getter so it's implicit.